docs/docs/welcome.md

```diff
@@ -4,35 +4,159 @@ title: Welcome to Botonic Documentation
+
+**With Botonic is a matter of adding a few lines of code:**
+
+```javascript
+export default class extends React.Component {
+  static async botonicInit({ input, session, params, lastRoutePath }) {
+    await humanHandOff(session))
+  }
+
+  render() {
+    return (
+      <Text>
+        Thanks for contacting us! One of our agents
+        will attend you as soon as possible.
+      </Text>
+    )
+  }
+}
+```
+
+Check out our [human handoff guide](/docs/concepts/humanhandoff).
+
+### Plugins
+
+Botonic comes with a battery of plugins so you can easily integrate popular services into your project, for instance:
+
+- **Analytics** ([Google Analytics](/docs/plugins/plugin-google-analytics), [Segment](/docs/plugins/plugin-segment), [Dashbot](/docs/plugins/plugin-dashbot))
+- **CMS** ([Contentful](/docs/plugins/plugin-contentful), Sanity)
+- **NLU** ([Dialogflow](/docs/plugins/plugin-dialogflow), [Watson](/docs/plugins/plugin-watson), [Luis](/docs/plugins/plugin-luis), [Inbenta](/docs/plugins/plugin-inbenta))
+
+You can also create your own plugins, just follow [our guide](/docs/plugins/createname-plugin1).
+
+### CLI
+
+We offer a command line interface tool that allows you to:
+
+- **Scaffold** new projects with `botonic new <botName> <template>`
+- **Run** your project locally with `botonic serve`
+- **Deploy** your project with `botonic deploy`
+
+### Typescript Support
+
+You can build your bot either with plain Javascript or with Typescript. **We provide TS typings!**
+
+Typescript examples coming soon...
+
+## Alternatives
+
+Botonic is an open-source alternative to...
+
+...Live-chat propietary tools:
+
+- [Intercom](https://www.intercom.com/)
+- [Liveperson](https://www.liveperson.com/)
+- [Freshchat](https://www.freshworks.com/live-chat-software/)
+- [Drift](https://www.drift.com/)
+- [Landbot](http://landbot.io/)
+- [Olark](https://www.olark.com/)
+
+...Messaging channels aggregators like:
+
+- [Zendesk Sunshine](https://www.zendesk.com/platform/conversations/outbound-messaging/) (formerly [Smooch](https://smooch.io/))
+
+...NLU closed services like:
+
+- [Dialogflow](https://cloud.google.com/dialogflow)
+- [Watson](https://www.ibm.com/watson)
+- [Luis](https://www.luis.ai/)
+- [Inbenta](https://www.inbenta.com/en/)
+
+Botonic's goal is not to fully replace all these amazing products feature by feature, but to provide a free and open-source option that covers most use cases. Also, we want to give you freedom to choose your own stack, by providing plugins to integrate many of these services in your project: want to use Botonic components but Dialogflow NLU? Easy, just add our [@botonic/plugin-dialogflow](/docs/plugins/plugin-dialogflow) and you're done.
```

Comment:
Also, we want -> We also want to... 
You want to use Dialogflow NLU? That's easy, just add our [@botonic/plugin-dialogflow](/docs/plugins/plugin-dialogflow) and you're done.